extend/browsing-downloading-from-extend.md

```diff
@@ -10,66 +10,78 @@ categories: [extend]
 
 As a user you're simply looking to download something: a plugin, a theme, an icon set, perhaps a language pack or just some other resource that you may find interesting.
 
-For that reason, you can quickly navigate from the menu to one of the 5 main categories:
+For that reason, you can quickly navigate from the menu to one of the 4 main categories:
 
-- Plugins & Modifications
+- Plugins
 
 - Themes
 
-- Graphics
-
-- Resources
+- Translations
 
-- Language Packs
+- Graphics
 
 Each main category has sub-categories, each with filtering options.
 
 
 ### Browsing a category
 
-[![Browsing and Filtering](/assets/images/mods/downloaders1.png)](/assets/images/mods/downloaders1.png)
+When you visit a category page, here's how it looks like:
+
+[![Browsing and Filtering](/assets/images/extend/downloaders1.png)](/assets/images/extend/downloaders1.png)
+
+For each projects you can view the cover and this informations:
```

Comment:
This doesn't make sense. I'd put- for each project you can view the cover and the following information: